code/game/objects/items.dm

```diff
@@ -129,6 +129,16 @@ GLOBAL_DATUM_INIT(welding_sparks_prepdoor, /mutable_appearance, mutable_appearan
 
 	var/active = FALSE
 
+	//Trick vars
+	///Sound played when you spin the item like the cool dude you are
+	var/spin_sound = 'sound/effects/spin.ogg'
+	///Sound played when you catch your item after a cool trick
+	var/thud_sound = 'sound/effects/thud.ogg'
+	///Delay between tricks
+	var/trick_delay = 6
+	///Time of last trick
+	var/recent_trick //So they're not spamming tricks.
```

Comment:
this could be a timer cooldown but i wont force you

Reply:
what's the advantage of a timer here?

Reply:
Doesnt need a var on item

Reply:
See TIMER_COOLDOWN or so